lib/table_of_contents/parser.rb

```diff
@@ -45,6 +62,7 @@ def parse_content
                .downcase
                .gsub(PUNCTUATION_REGEXP, '') # remove punctuation
                .tr(' ', '-') # replace spaces with dash
+          id = url_encode(id)
```

Comment:
Could you file a issue ❓ 

> `äöü` doesn't work for id

Reply:
Of course, I created the Issue: https://github.com/toshimaru/jekyll-toc/issues/95

Reply:
Thanks for creating an issue :)

Could you separate PR for fixing #95 . This helps me to review PR. :)

Reply:
Thanks for creating an issue :)

Could you separate PR for fixing #95 . This helps me to review PR. :)

Reply:
You mean split the PR one for navigate to top / toc and a second PR for url_encoding?

Reply:
@N7K4
Yes. please separate this commit 💨 .  And one more thing, could you add test for `url_encoding ` ❓ 😉 

> The URL_code was just the first commit: 1f0be2f

Reply:
https://github.com/toshimaru/jekyll-toc/pull/101